docker/cross-files/loongarch64-gcc-14-ccache.cross

```diff
@@ -0,0 +1,18 @@
+[binaries]
+c = ['ccache', 'loongarch64-unknown-linux-gnu-gcc']
+cpp = ['ccache', 'loongarch64-unknown-linux-gnu-g++']
+ar = 'loongarch64-unknown-linux-gnu-ar'
+strip = 'loongarch64-unknown-linux-gnu-strip'
+objcopy = 'loongarch64-unknown-linux-gnu-objcopy'
+ld = 'loongarch64-unknown-linux-gnu-ld'
+exe_wrapper = ['qemu-loongarch64-static', '-L', '/opt/cross-tools/target/usr']
+
+[properties]
```

Comment:
Should add '-mlsx' to compile LSX builtin code.

Reply:
Is that a feature of all loongson CPUs?